docs/docs-content/enterprise-version/install-palette/install-on-vmware/vmware-system-requirements.md

```diff
@@ -0,0 +1,300 @@
+---
+sidebar_label: "VMware System and Permission Requirements"
+title: "VMware System and Permission Requirements"
+description: "Review VMware system requirements and cloud account permissions."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["palette", "self-hosted", "vmware"]
+---
+
+
+Before installing Palette on VMware, review the following system requirements and permissions. The vSphere user account used to deploy Palette must have the required permissions to access the proper roles and objects in vSphere. 
+
+Start by reviewing the required action items below:
+
+1. Create the two custom vSphere roles. Check out the [Create Required Roles](#create-required-roles) section to create the required roles in vSphere. 
+
+2. Review the [vSphere Permissions](#vsphere-permissions) section to ensure the created roles have the required vSphere privileges and permissions. 
+
+3. Create node zones and regions for your Kubernetes clusters. Refer to the [Zone Tagging](#zone-tagging) section to ensure that the required tags are created in vSphere to ensure proper resource allocation across fault domains.
+
+
+:::info
+
+The permissions listed in this page are also needed for deploying a Private Cloud Gateway (PCG), and workload cluster in vSphere through Palette.
+:::
+
+
+## Create Required Roles
+
+Palette requires two custom roles to be created in vSphere before the installation. Refer to the [Create a Custom Role](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html?hWord=N4IghgNiBcIE4HsIFMDOIC+Q) guide if you need help creating a custom role in vSphere. The required custom roles are:
+
+* A root-level role with access to higher-level vSphere objects. This role is referred to as the *spectro root role*. Check out to the [Root-Level Role Privileges](#root-level-role-privileges) table for the list of privileges required for the root-level role.
+
+* A role with the required privileges for deploying VMs. This role is referred to as the *Spectro role*. Review the [Spectro Role Privileges](#spectro-role-privileges) table for the list of privileges required for the Spectro role. 
+
+
+The user account you use to deploy Palette must have access to both roles. Each vSphere object required by Palette must have a [Permission](https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.security.doc/GUID-4B47F690-72E7-4861-A299-9195B9C52E71.html) entry for the respective Spectro role. The following tables list the privileges required for the each custom role. 
+
+
+
+
+:::info
+
+For an-idepth overview of vSphere authorization and permissions overview, check out the [Understanding Authorization in vSphere](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-74F53189-EF41-4AC1-A78E-D25621855800.html) resource.
```

Comment:
Is it an "in-depth" (typo: an-idepth) or an "overview"? Maybe the following is intended?